member/src/main/java/store/_0982/member/presentation/MemberController.java

```diff
@@ -0,0 +1,75 @@
+package store._0982.member.presentation;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+import store._0982.member.application.MemberService;
+import store._0982.member.application.dto.*;
+import store._0982.member.common.dto.ResponseDto;
+import store._0982.member.presentation.dto.MemberSignUpRequest;
+import store._0982.member.presentation.dto.PasswordChangeRequest;
+import store._0982.member.presentation.dto.ProfileUpdateRequest;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@RestController
+@RequestMapping("/api/members")
+@RequiredArgsConstructor
+@Slf4j
+public class MemberController { //todo log 달기
+    private final MemberService memberService;
+    @PostMapping
+    public ResponseDto<MemberSignUpInfo> signUpMember(@RequestBody MemberSignUpRequest memberSignUpRequest) {
+        MemberSignUpInfo memberSignUpInfo = memberService.signUpMember(memberSignUpRequest.toCommand());
+        return new ResponseDto<>(HttpStatus.OK, memberSignUpInfo, "회원가입이 완료되었습니다.");
+    }
+
+    @DeleteMapping
+    public ResponseDto<Object> deleteMember(@RequestHeader(value = "X-Member-Id", required = false) String memberId , @RequestBody MemberDeleteRequest request){
+        memberService.deleteMember(new MemberDeleteCommand(memberId, request.password()));
+        return new ResponseDto<>(HttpStatus.OK, null, "회원가입이 완료되었습니다.");
```

Comment:
```suggestion
        return new ResponseDto<>(HttpStatus.OK, null, "회원 탈퇴가 완료되었습니다.");
```
회원 탈퇴 기능인 것 같은데 메시지가 수정되어야 할 것 같습니다!